lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -58,6 +58,7 @@ WebOb = "*"
 Mako = "*"
 Babel = "*"
 Beaker = "*"
+diskcache = "*"
```

Comment:
Can you also remove dogpile.cache from here and `lib/galaxy/dependencies/pipfiles/default/pinned-requirements` ?

Reply:
Sure!

Reply:
Actually, I think it might be, we've had dog.pile in our dependencies before I started using it for the document cache (https://github.com/galaxyproject/galaxy/blob/release_20.01/lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt#L55)

Reply:
OK, but you can still remove it from this file (`Pipfile`), which lists our explicit dependencies.

Reply:
Ah, it should have been there, but you're right it's not, so nothing to do!

Reply:
This is not used any more, should be replaced by `sqlitedict`.